main.go

```diff
@@ -2275,25 +2288,192 @@ The following options are recognized:
 				return err
 			}
 
-			s, err := ca.Secret(opt.SkipIfExists)
+			err = ca.SaveTo(v, opt.X509.Issue.SignedBy, opt.SkipIfExists)
+			if err != nil {
+				return err
+			}
+		}
+
+		err = cert.SaveTo(v, args[0], opt.SkipIfExists)
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+
+	r.Dispatch("x509 reissue", &Help{
+		Summary: "Reissue X.509 Certificates and Certificate Authorities",
+		Usage:   "safe x509 reissue [OPTIONS] path/to/certificate",
+		Description: `
+Reissues an X.509 Certificate with a new key.
+
+The following options are recognized:
+
+  -b, --bits  N      RSA key strength, in bits.  The only valid
+                     arguments are 1024 (highly discouraged),
+                     2048 and 4096.  Defaults to 4096
+
+  -i, --signed-by   Path in the Vault where the CA certificate
+                    (and signing key) can be found.  If this is not
+                    provided, a sibling secret named 'ca' will used
+                    if it exists.
```

Comment:
Might want to note that this _should_ be the same as the original signing CA, but need not be.